backend/telemetry_core/src/state/chain_stats.rs

```diff
@@ -96,6 +96,27 @@ fn bucket_memory(memory: u64) -> (u32, Option<u32>) {
         48,
         56,
         64,
+        128,
+    }
+}
+
+fn kernel_version_number(version: &Box<str>) -> &str {
+    &version[0..version.find('-').unwrap_or(version.len())]
+}
+
+#[test]
+fn test_kernel_version_number() {
+    assert_eq!(kernel_version_number(&"5.10.0-8-amd64".into()), "5.10.0");
+    assert_eq!(kernel_version_number(&"5.10.0".into()), "5.10.0");
+}
+
+fn cpu_vendor(cpu: &Box<str>) -> &str {
+    if cpu.contains("Intel") || cpu.contains("intel") {
```

Comment:
maybe https://doc.rust-lang.org/std/string/struct.String.html#method.eq_ignore_ascii_case to keep the code a little simpler?